php/libraries/BVL_Feedback_Panel.class.inc

```diff
@@ -33,9 +33,16 @@ class BVL_Feedback_Panel
     /**
      * The feedback thread that this panel is for
      *
-     * @var NDB_BVL_Feedback
+     * @private NDB_BVL_Feedback
      */
-    var $_feedbackThread;
+    private $_feedbackThread;
```

Comment:
```suggestion
    protected $_feedbackThread;
```

protected is more appropriate here, any (theoretical) class that derives from this would need access to it.